### PR TITLE
Added accounts query and tests

### PR DESCRIPTION
**Pull request by Kayanski**
This Pr aims at 
- changing the `Account` query on the registry contract to an `Accounts` query
- Adding an paginated `AccountList` query on the registry contract

### Checklist

- [ ] CI is green.
- [x] Changelog updated.

### CODE REVIEW

**Review by CyberHoward — APPROVED** [review]

File: framework/contracts/account/tests/create.rs [hunk 1]
```diff
@@ -78,9 +78,9 @@ fn create_one_account() -> AResult {
 
     assert_that!(&vc_config).is_equal_to(&expected);
 
-    let account_list = registry.account(TEST_ACCOUNT_ID)?;
+    let account_list = registry.get_account(TEST_ACCOUNT_ID)?;
```
Comment (Buckram123) [inline framework/contracts/account/tests/create.rs:81]: Are we keeping `get_` prefix for the interface?

**Review by Buckram123 — APPROVED** [review]
LGTM

File: framework/contracts/account/tests/create.rs (hunk 1 above)
Comment (CyberHoward) [reply framework/contracts/account/tests/create.rs:81]: we could do just `account` as we changed the name of the actual query to `accounts`

Comment (CyberHoward) [reply framework/contracts/account/tests/create.rs:81]: renamed it

**Pull request merged**